scripts/fuzz.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e
+
+function build_fuzz() {
+  PACKAGES=$1
+  
+  for pkg in $PACKAGES; do
+    pushd fuzz/$pkg
+  
+    for file in *.go; do
+      if [[ "$file" == 'fuzz_utils.go' ]]; then
+        continue
+      fi
+  
+      NAME=$(echo $file | sed 's/\.go$//1')
+      echo "Building zip file for $pkg/$NAME"
+      go-fuzz-build -func "Fuzz_$NAME" -o "$pkg-$NAME-fuzz.zip" "github.com/lightningnetwork/lnd/fuzz/$pkg"
+    done
+  
+    popd
+  done
+}
+
+# timeout is a cross platform alternative to the GNU timeout command that
+# unfortunately isn't available on macOS by default.
+timeout() {
+    time=$1
+    $2 &
+    pid=$!
+    sleep $time
+    kill -s SIGINT $pid    
+}
+
+function run_fuzz() {
+  PACKAGES=$1
+  RUN_TIME=$2
+  TIMEOUT=$3
+  PROCS=$4
+  
+  for pkg in $PACKAGES; do
+    pushd fuzz/$pkg
+    DIR=$(pwd)
+  
+    for file in *.go; do
+      if [[ "$file" == 'fuzz_utils.go' ]]; then
+        continue
+      fi
+  
+      NAME=$(echo $file | sed 's/\.go$//1')
+      echo "Running fuzzer $pkg-$NAME-fuzz.zip with $PROCS processors for $RUN_TIME seconds"
+      COMMAND="go-fuzz -bin=$pkg-$NAME-fuzz.zip -workdir=$DIR -procs=$PROCS -timeout=$TIMEOUT"
```

Comment:
`-workdir` should be fuzzer-specific so something like `$DIR/$NAME`.
If we start mixing for example the `lnwire` inputs all together, then the fuzzing engine will be less effective at figuring out the parsing format of a particular message.

Reply:
Ah, didn't know that. Fixed.

Reply:
You can see how I organize them locally here: https://github.com/Crypt-iQ/lnd_fuzz_seeds
There's a bit of background on why to separate them (including links to relevant bitcoin core prs) here https://github.com/lightningnetwork/lnd/pull/1895

Reply:
Soooo, what's the diff to what I just implemented now?
Looks like this on my machine now:
```text
fuzz/wtwire
├── create_session
│   ├── corpus
│   │   ├── 48899077e8a88ff3d3f2a9d38724b0cbdfbfdf5a-2
│   │   ├── 69d2ef40f4ac35b7b8d0d81e9030dde33e4a7820-1
│   │   ├── da39a3ee5e6b4b0d3255bfef95601890afd80709
│   │   ├── de7c872871483a994565e03fe78a77121ebf86d8-2
│   │   └── e01a1822dfae1c6b16a70479ed0c9c4d28cd2344-1
│   ├── crashers
│   └── suppressions
```

Reply:
If the fuzzers have nothing to start with in the work directory, it won't work as well if we give it the data to start with. It will still generate inputs, but it will have a better time if we start with the data.